Frends.JSON.ConvertXMLStringToJToken/Frends.JSON.ConvertXMLStringToJToken/Definitions/Input.cs

```diff
@@ -10,4 +10,12 @@ public class Input
     /// </summary>
     /// <example>&lt;?xml version='1.0' standalone='no'?&gt;&lt;root&gt;&lt;foos id = '1' &gt;&lt;foo&gt;bar&lt;/name&gt;&lt;/foos&gt;&lt;/root&gt;</example>
     public string XML { get; set; }
+
+    /// <summary>
+    /// A list of XPath expressions that specify which XML elements should always be converted to arrays.
```

Comment:
So those are not really XPath :) Those are JSONPath you have here. So lets change the docs to say "...JSONPath expressions that specify which JSON elements..."